scripts/spell-check.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+## Copyright (C) 2020 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# This script runs codespell against selected files.
+#
+# Use -q or --quiet option to run quietly
+#
+
+CODESPELL_LOC=`which codespell`
+if test "${CODESPELL_LOC}" = ""; then
+    echo "This script requires codespell which was not found."
+    exit 1
+fi
+
+CODESPELL_VER=`codespell --version 2>&1`
+echo "The codespell version is ${CODESPELL_VER}."
+
+UNSTAGED_CHANGES=`git diff | wc -l`
+if test "${UNSTAGED_CHANGES}" != "0"; then
+    echo "There are unstaged changes. Stage these first to prevent conflict."
+    exit 1
+fi	
+
+WHITE_LIST=scripts/codespell-whitelist.txt
+if test ! -f "${WHITE_LIST}"; then
+    echo "${WHITE_LIST} does not exist"
+    exit 1
+fi
+
+QUIET=0
+while test "$1" != ""; do
+    case $1 in
+        -q | --quiet )
+            QUIET=1
+            ;;
+    esac
+    shift
+done
+
+#
+# Scan for file-specific actions
```

Comment:
Stale/copied comment?

Reply:
This was copied from another script but seemed appropriate.
Removed in next commit.